acceptance_tests_pod.yml

```diff
@@ -0,0 +1,162 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: acceptance-tests
+  labels:
+    app: acceptance-tests
+    env: dev
```

Comment:
We got rid of these env labels we aren't using. Instead, however, it would be good to tag it with it's dependencies like we've started going with the apps.